packages/ilios-common/addon/components/offering-manager.gjs

```diff
@@ -40,33 +42,63 @@ export default class OfferingManagerComponent extends Component {
     return new TrackedAsyncData(this.args.offering.learnerGroups);
   }
 
-  @cached
-  get sessionData() {
-    return new TrackedAsyncData(this.args.offering?.session);
+  get learnerGroups() {
+    return this.learnerGroupsData.isResolved ? this.learnerGroupsData.value : null;
   }
 
-  @cached
-  get courseData() {
-    return new TrackedAsyncData(this.session?.course);
+  get sortedLearnerGroups() {
+    if (!this.learnerGroups) {
+      return [];
+    }
+    return this.learnerGroups.slice().sort((learnerGroupA, learnerGroupB) => {
+      const locale = this.intl.get('locale');
+      if ('title:desc' === this.sortBy) {
+        return learnerGroupB.title.localeCompare(learnerGroupA.title, locale, { numeric: true });
```

Comment:
```suggestion
        return learnerGroupB.title.localeCompare(learnerGroupA.title, this.intl.primaryLocale, { numeric: true });
``` don't need the `get` on locale anymore, grab it like this for both of these instead of storing the interim value.